pkg/scheduler/framework/plugins/loadaware/load_aware.go

```diff
@@ -1,19 +1,3 @@
-/*
```

Comment:
why do you delete this ?

Reply:
It is a misoperation, and all headers have been restored